<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub_data_table</class>
 <widget class="QWidget" name="sub_data_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_data">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>931</width>
     <height>431</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>年龄</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>表情</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>male</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>15-20</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>neutral</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_start">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_end">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_start">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_end">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pub_find">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pub_prior">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pub_next">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
